src/Cli/Action/AwsClientFactory.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AcmePhp\Cli\Action;
```

Comment:
Perhaps a dedicated `Aws` namespace would be more appropriate?

Reply:
:+1: